prettyfonts/v11/v11-PyQt5.py

```
@@ -1908,6 +2511,85 @@ def on_update_check_completed(self, release_info: dict):
             QMessageBox.information(self, self.tr("Update"), self.tr("New version available: %s") % tag)
         else:
             QMessageBox.information(self, self.tr("Update"), self.tr("No updates available."))
+    
+    def open_encryption_tool(self):
+        dlg = TextEncToolDialog(self)
```

Ruff (0.12.2)

Undefined name `TextEncToolDialog`

(F821)